openjdk-11.yaml

```diff
@@ -10,6 +10,10 @@ package:
         - "*"
       attestation:
       license: GPL-2.0-only
+  dependencies:
+    runtime:
+      - freetype-dev
```

Comment:
However, freetype-dev and fontconfig-dev are runtime dependencies of the JDK?  Why is that?  The runtime depends will get detected automatically by Melange, or should be.

Reply:
I'll double check this in the morning. I was seeing a few instances of
```
libfontmanager.so: libfreetype.so.6: cannot open shared object file: No such file or directory
```
Though I now wonder if that was the JDK or the JRE subpackage.  I'll do a test and make sure.

Reply:
Just ran a test removing these runtime dependencies and get the error I was mentioning above:
```
LoadStatisticsTest.graph:94 » UnsatisfiedLink /usr/lib/jvm/openjdk/lib/libfontmanager.so: libfreetype.so.6: cannot open shared object file: No such file or directory
```

Reply:
Just testing replacing the runtime dependencies using
```
      - freetype
      - fontconfig
```
rather than
```
      - freetype-dev
      - fontconfig-dev
```

Reply:
I've ran some downstream tests and this is working.